SLURM_PHOENIX_README.md

```diff
@@ -0,0 +1,28 @@
+# Running Conformance Tests on the Phoenix SLURM Cluster
+This is an addendum to [`SLURM_README.md`](SLURM_README.md) and contains some small notes on how to run conformance tests for the Phoenix cluster.
+
+For Singularity, the Singularity cache directories must be configured [as mentioned](SLURM_README.md#singularity-cache).
+## Under Cromwell
+No extra configuration should be necessary.
+
+## Under MiniWDL
+Additional note to:
+```
+However, if running the script under an srun launched interactive terminal, this will not work.
+
+This is likely because `miniwdl_slurm` runs [srun itself](https://github.com/miniwdl-ext/miniwdl-slurm/blob/624ab390ea872082798733fefbb327dec99e2cde/src/miniwdl_slurm/__init__.py#L97-L100) instead of sbatch, and nested srun's dont appear to work (likely something related to the nested srun asking for more resources beyond the scope of the parent srun).
+```
+On Phoenix, this can be avoided by running the scripts from mustard, emerald, crimson, razzmatazz. (The Phoenix head node can also be used, but this is not probably not recommended.)
```

Comment:
```suggestion
On Phoenix, this can be avoided by running the scripts from mustard, emerald, crimson, or razzmatazz. (The Phoenix head node can also be used, but this is not probably not recommended.)
```